index.html

```diff
@@ -198,6 +197,56 @@ <h3 class="skill-cat">Skills</h3>
       </ul>
     </section>
   </main>
+  <footer>
+    <section class="contact-section">
+      <h2 class="get-started">Get started</h2>
```

Comment:
- Please i suggest in your h2 element you would have used a class-like heading-2 for someone to understand your code easily while styling.